files/zh-cn/mdn/community/contributing/our_repositories/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: MDN Web 文档仓库
+slug: MDN/Community/Contributing/Our_repositories
+---
+
+{{MDNSidebar}}
+
+[MDN Web 文档](/)是一个复杂的项目，有很多不同的部分。熟悉项目的不同仓库（repository）是个好主意。本文档旨在帮助你找到在为 MDN Web 文档项目中不同部分做出贡献时可能需要的不同仓库。
```

Comment:
Should we redirect to localized slug?

```suggestion
[MDN Web 文档](/zh-CN)是一个复杂的项目，有很多不同的部分。熟悉项目的不同仓库（repository）是个好主意。本文档旨在帮助你找到在为 MDN Web 文档项目中不同部分做出贡献时可能需要的不同仓库。
```

Reply:
This would automatically redirected to zh-CN, it's ok to leave it as it is (with the English document)